### PR TITLE
language-clojure-update

### PR DESCRIPTION
**Pull request by thomasr8**
# Description
Replaced legacy images with next-gen images

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

**Comment by devinmarieb** [comment]
Clojure has no /cimg yet

File: jekyll/_cci2/language-clojure.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs: # basic units of work in a run
   build: # runs not using Workflows must have a `build` job as entry point
     working_directory: ~/cci-demo-clojure # directory where steps will run
     docker: # run the steps with Docker
-      - image: circleci/clojure:lein-2.9.1 # ...with this image as the primary container; this is where all `steps` will run
+      - image: cimg/clojure:lein-2.9.1 # ...with this image as the primary container; this is where all `steps` will run
```
Comment (felicianotech) [inline jekyll/_cci2/language-clojure.md:43]: The tag `lein-2.9.1` doesn't exist. We can use `1.10`.

File: jekyll/_cci2/language-clojure.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ Directly beneath `working_directory`, we can specify container images under a `d
 version: 2
 ...
     docker:
-      - image: circleci/clojure:lein-2.9.1
+      - image: cimg/clojure:lein-2.9.1
```
Comment (felicianotech) [inline jekyll/_cci2/language-clojure.md:105]: The tag `lein-2.9.1` doesn't exist. We can use `1.10`.

**Review by felicianotech — CHANGES_REQUESTED** [review]
I know this PR is in draft and that next-gen Clojure is in beta, reviewing anyway.

File: jekyll/_cci2/language-clojure.md (hunk 1 above)
Comment (thomasr8) [reply jekyll/_cci2/language-clojure.md]: Fixed

File: jekyll/_cci2/language-clojure.md (hunk 2 above)
Comment (thomasr8) [reply jekyll/_cci2/language-clojure.md]: Fixed

**Review by felicianotech — APPROVED** [review]

**Pull request merged**